### PR TITLE
[SMALLFIX] Spelling/grammar in FAQ

### PR DESCRIPTION
**Pull request by aurnik**
Small changes to FAQ page in project documentation

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -16,31 +16,31 @@ group: FAQ
...
 
->   No, Currently re-balancing of data blocks in not available.
+>   No, currently re-balancing of data blocks is not available.
 
 **Is Tachyon master fault tolerant?**
 
 >   Yes. Please see this [page](http://tachyon-project.org/Fault-Tolerant-Tachyon-Cluster.html).
 
-**What is under FS and how many Underlayer FS are supported?**
+**What is UnderFS and how many UnderlayerFS are supported?**
```
Comment (apc999) [inline docs/FAQ.md]: not sure if `UnderlayerFS` is a term in Tachyon world. Maybe just `UnderFS`?

File: docs/FAQ.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ newly added nodes in order to balance disk space utilization between the nodes?*
 
 **Can I run Tachyon as a standalone for experimenting and validation?**
 
->    Yes, Tachyon run in a stand-alone mode. Instructions are available
+>    Yes, Tachyon can run in a stand-alone mode. Instructions are available
```
Comment (gpang) [inline docs/FAQ.md]: "in a stand-alone mode" -> "in stand-alone mode"

File: docs/FAQ.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ newly added nodes in order to balance disk space utilization between the nodes?*
 
 **Can I run Tachyon as a standalone for experimenting and validation?**
```
Comment (gpang) [inline docs/FAQ.md]: can you change this to "Can I run Tachyon in stand-alone mode ..."

**Comment by gpang** [comment]
Thanks for the improvements! I left a few minor comments.

File: docs/FAQ.md (hunk 1 above)
Comment (aurnik) [inline docs/FAQ.md]: @apc999 So would the correction be "What is UnderFS and how many UnderFS are supported?"

File: docs/FAQ.md (hunk 3 above)
Comment (aurnik) [inline docs/FAQ.md]: Fixed

File: docs/FAQ.md (hunk 2 above)
Comment (aurnik) [inline docs/FAQ.md]: Fixed

File: docs/FAQ.md (hunk 1 above)
Comment (haoyuan) [inline docs/FAQ.md]: yes.

Comment (aurnik) [inline docs/FAQ.md]: Fixed

**Comment by gpang** [comment]
LGTM

**Comment by apc999** [comment]
+1 LGTM, thanks for the patch!

**Pull request merged**